src/themes/default/components/core/blocks/Wishlist/Wishlist.vue

```diff
@@ -5,9 +5,12 @@
         <i class="material-icons p15 pointer cl-accent" @click="closeWishlist">close</i>
       </div>
     </div>
-    <h2 v-if="productsInWishlist.length" class="cl-accent ml30">
-      {{ $t('Wishlist') }}
-    </h2>
+    <div class="row middle-xs px40">
+      <h2 v-if="productsInWishlist.length" class="col-xs-12 col-sm cl-accent">
+        {{ $t('Wishlist') }}
+      </h2>
+      <clear-wishlist-button v-if="productsInWishlist.length" @click.native="clearWishlist" class="col-xs-12 col-sm mt35 mb35 end-sm"/>
```

Comment:
```suggestion
      <clear-wishlist-button v-if="productsInWishlist.length" @click="clearWishlist" class="col-xs-12 col-sm mt35 mb35 end-sm"/>
```
we should avoid using `native` - if it's our component button should emit click event

Reply:
done